verification/src/transaction_verifier.rs

```diff
@@ -221,14 +221,15 @@ impl<'a> MaturityVerifier<'a> {
 
     pub fn verify(&self) -> Result<(), TransactionError> {
         let cellbase_immature = |meta: &CellMeta| -> bool {
-            meta.is_cellbase()
-                && self.block_number
-                    < meta
-                        .transaction_info
-                        .as_ref()
-                        .expect("cell meta should have block number when transaction verify")
-                        .block_number
-                        + self.cellbase_maturity
+            let cell_block_number = meta
+                .transaction_info
+                .as_ref()
+                .expect("cell meta should have block number when transaction verify")
+                .block_number;
+            // Ignore genesis cellbase maturity check
```

Comment:
```suggestion
            // Ignore genesis cells maturity check
```

Reply:
`maturity` is a `cellbase` only property, why `cells`?

Reply:
Yes, use `cellbase` here is also makes sense.